ui/login.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
```

Comment:
This template can be replaced with webpack's htmltemplate once this is pure React (see the test-view's template config in `neutrino-custom/base.js` for an example)